plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -843,22 +880,27 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
         interpolator = it
       }
       animationOptions?.animatorListener?.let {
-        addListener(object : AnimatorListenerAdapter() {
-          override fun onAnimationEnd(animation: Animator) {
-            commitChanges()
+        // listeners in Android SDK use non thread safe lists
+        postOnAnimatorThread {
+          addListener(object : AnimatorListenerAdapter() {
```

Comment:
maybe simplify because you've got only one function inside?

Reply:
don't get how to simplify it